casbin_adapter/adapter.py

```diff
@@ -2,17 +2,21 @@
 from casbin import persist
 
 from .models import CasbinRule
+from django.db.utils import OperationalError, ProgrammingError
 
 
 class Adapter(persist.Adapter):
     """the interface for Casbin adapters."""
 
     def load_policy(self, model):
         """loads all policy rules from the storage."""
-        lines = CasbinRule.objects.all()
+        try:
+            lines = CasbinRule.objects.all()
 
-        for line in lines:
-            persist.load_policy_line(str(line), model)
+            for line in lines:
+                persist.load_policy_line(str(line), model)
+        except (OperationalError, ProgrammingError):
+            pass
```

Comment:
The error should not be passed, if it is to be caught, is it possible to log it?

Reply:
ok. changed, set to warn because it's raised in django-authorization when django initial(will reload later)